### PR TITLE
Add support for Docker secrets

### PR DESCRIPTION
**Pull request by mathieupotier**
Add support for docker secrets : https://docs.docker.com/engine/swarm/secrets/

Created new environment var to manage secrets source files.

Hope it helps, made such a change for my usage, and seems working perfectly ^_^

### CODE REVIEW

**Comment by J0WI** [comment]
I'd prefer to use the same pattern as [other php images](https://github.com/docker-library/wordpress/blob/5125491823651bcd691faf415d836f911fea09cd/docker-entrypoint.sh#L4-L24) do. Would you like to implement this?

**Comment by mathieupotier** [comment]
I can do it, I'll push this in few hours.

**Comment by mathieupotier** [comment]
Done.

I just prefered to not export env var to avoid leaks on secrets.
The replaced env vars are just shared under current PID, they don't go to the kernel stack.

File: README.md [hunk 1]
```diff
@@ -66,6 +66,19 @@ You can also set the postfix host and port.
...
+As an alternative to passing sensitive information via environment variables, `_FILE` may be appended to some of the previously listed environment variables, causing the initialization script to load the values for those variables from files present in the container. In particular, this can be used to load passwords from Docker secrets stored in /run/secrets/<secret_name> files. For example:
+
+```bash
+docker run -e POSTFIXADMIN_DB_USER_FILE=/run/secrets/postfix-db-user
+           -e POSTFIXADMIN_DB_PASSWORD_FILE=/run/secrets/postfix-db-passwd
+           -e POSTFIXADMIN_SETUP_PASSWORD_FILE=/run/secrets/postfix-setup-passwd
+        postfixadmin-image
+```
+
+Currently, this is only supported for `POSTFIXADMIN_DB_USER`, `POSTFIXADMIN_DB_PASSWORD`, and `POSTFIXADMIN_SETUP_PASSWORD`.
```
Comment (mathieupotier) [inline README.md:80]:
```suggestion
All environment vars are supporting the secret's docker strategy.
```

**Review by J0WI — APPROVED** [review]
Not sure if anyone needs a secret for port or host, but LGTM. Thanks!

**Pull request merged**